locations/spiders/basilic_and_co_fr.py

```
@@ -0,0 +1,35 @@
+
+class BasilicAndCoSpider(SitemapSpider, StructuredDataSpider):
+    name = "basilic_and_co"
+    item_attributes = {
+        "brand": "Basilic & Co",
+        "brand_wikidata": "Q130248490",
+    }
+    sitemap_urls = ["https://pizzerias.basilic-and-co.com/robots.txt"]
+    sitemap_rules = [
+        (r"/restaurant-pizza-terroir/", "parse_sd"),
+    ]
+    drop_attributes = {"image", "facebook", "name"}
```

Ruff (0.16.1)

[warning] Mutable default value for class attribute

(RUF012)